java/src/jmri/jmrix/pi/ConnectionConfig.java

```diff
@@ -94,10 +94,6 @@ public boolean getDisabled(){
     public void setDisabled(boolean disable){
        this.disabled=disable;
     }
-
-   @Override
-   public void dispose(){
```

Comment:
On Nov 27, 2015, at 9:46 AM, Randall Wood <notifications@github.com> wrote:
> In java/src/jmri/jmrix/pi/ConnectionConfig.java:
> 
> > @@ -94,10 +94,6 @@ public boolean getDisabled(){
> >      public void setDisabled(boolean disable){
> >         this.disabled=disable;
> >      }
> > -
> > -   @Override
> > -   public void dispose(){
> This is the only instance of dispose() in a ConnectionConfig that did not call super.
> 
> @pabender Did this intentionally not call super.dispose()?
> 

Not intentionally.  I am sure it was copied from some other file as a starting point though.

Paul